stock_picking_progress/models/stock_move.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+from odoo.tools.float_utils import float_is_zero
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    progress = fields.Float(
```

Comment:
That make me rethink about what we said regarding the computation of the `stock.picking.progress` field: IMO it should be computed from `stock.move.progress` instead of `stock.move.line.progress` so we include qty that can't be reserved (no `stock.move.line` created).

Should it be a drawback for Shopfloor or not?

Reply:
My take: `picking.progress` field should be computed from the moves if its delivery policy is "all at once", and from move lines if the policy is "as soon as possible". That way we got results that the user could expect.
ping @jbaudoux 
I won't block for now as this PR doesn't change the computation at the picking level but it's something we need to check later.